stdpopsim/catalog/MusMus/species.py

```diff
@@ -138,15 +138,19 @@
 # two values
 
 # Estimates of effective population size for M. musculus can vary quite a lot. This
-# summary is complicated further as the specific sub-species of M. musculus are described
-# as having different effective population sizes (though not, at least, of a different
-# order of magnitude). In the suggested citations the effective population sizes of
+# summary is complicated further as specific subspecies of M. musculus are described
+# as having different effective population sizes (see Fujiwara et al. (2022);
+# though not, at least, of a different order of magnitude).
+# In the suggested citations the effective population sizes of
```

Comment:
I changed this to the Phifer-Rixey, et al. (2012) citation, let me know if you think this should be replaced.

Reply:
I agree that Phifer-Rixey, et al. (2012) is a good publication to cite for ballpark estimates of Ne for different subspecies, since they mention these in the abstract.